mesheryctl/cmd/perf.go

```diff
@@ -175,5 +190,6 @@ func init() {
 	perfCmd.Flags().StringVar(&testDuration, "duration", "30s", "(optional) Length of test (e.g. 10s, 5m, 2h). For more, see https://golang.org/pkg/time/#ParseDuration")
```

Comment:
Note to me:-
Endpoint URL is still required PARAM, it should be optional, if file is supplied.